plugins/homepage/server/__init__.py

```diff
@@ -32,6 +37,7 @@ def __init__(self):
         super(Homepage, self).__init__()
         self.resourceName = 'homepage'
         self.route('GET', ('markdown',), self.getMarkdown)
+        self.route('GET', ('settings',), self.getSettings)
```

Comment:
There should be no need to define another new route. The new settings can be returned as additional fields in the response of the existing route.